### PR TITLE
Add a description about RLS to IVM documentation.

### PR DESCRIPTION
**Pull request by yugo-n**
Github issue #71

### CODE REVIEW

File: doc/src/sgml/rules.sgml [hunk 1]
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
 </para>
 </sect2>
 
+<sect2>
+<title>Row Level Security</title>
+<para>
+    If some base table have row level security policy, rows that are not invisible
```
Comment (tatsuo-ishii) [inline doc/src/sgml/rules.sgml]: "base table" should be "base tables".
"invisible for" should be "invisible to".

File: doc/src/sgml/rules.sgml [hunk 2]
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
...
+    for the materialized view are excluded from the result. In addition, such rows
+    are excluded also when views are incrementally maintained. However, if a new
```
Comment (tatsuo-ishii) [inline doc/src/sgml/rules.sgml]: "also" should "as well" or "too".

File: doc/src/sgml/rules.sgml [hunk 3]
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
...
+    policy is defined  or policies are changed after materialized view definition,
```
Comment (tatsuo-ishii) [inline doc/src/sgml/rules.sgml]: There's extra space after "defined".
"after materialized view definition" should be "after the materialized view definition was created"

File: doc/src/sgml/rules.sgml [hunk 4]
```diff
@@ -1528,6 +1528,18 @@ postgres=# EXPLAIN SELECT * FROM m1;
...
+    the new policy will not applied to the view contents. To apply the new policy,
```
Comment (tatsuo-ishii) [inline doc/src/sgml/rules.sgml]: "will not applied" should be "will not be applied".

**Review by tatsuo-ishii — CHANGES_REQUESTED** [review]

**Comment by yugo-n** [comment]
@tatsuo-ishii Thank you so much for your many pointing out my mistake!  I'll fix them.

**Comment by yugo-n** [comment]
@tatsuo-ishii 

I fixed the typos. Also, I added a missing word as following:

"not invisible to the materialized view" 
 -> "not invisible to the materialized view's owner"

**Review by tatsuo-ishii — APPROVED** [review]

**Comment by yugo-n** [comment]
Thanks for your review.  I'll merge it.

**Pull request merged**